### ISSUE TITLE
"Next Up" home row not updating

### ISSUE THREAD

**Issue by cewert** [jellyfin-archive/jellyfin-roku-legacy#1184]
Bug was introduced in #955

## Software Versions

- Jellyfin Server Version:  latest stable
- Roku Client Version:  v1.6.4

## Describe the bug
When returning to the home screen after exiting a video, the Next Up row is not being updated properly. The continue watching seems to still work as expected.

## Additional context
First reported on [reddit](https://old.reddit.com/r/jellyfin/comments/12crra5/jellyfin_for_roku_164_released/jfgakeb/) and also in the matrix chat

**Comment by azokthedefiler1** [comment]
I'm having the same issue and found this in my log files:

`jellyfin  | [02:54:07] [INF] [120] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app Jellyfin Roku 1.6.4 playing <FILE> Stopped at 1419000 ms
jellyfin  | [02:54:07] [INF] [107] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Playback stop did not have a tracker : <GUID>
`

May or not be helpful. Otherwise, everything else in my logs looks normal.

**Comment by p37307** [comment]
I am in that thread on Reddit. I've been doing some testing. 

After the Roku 1.6.4 upgrade, Next Up on the Home Screen is messed up. I tried this on various models of Roku.
When I watch a TV show on Roku 1.6.4, it marks the episode as watched but removes it from the Next Up section on the home screen on all devices and platforms.
So,  I log into Jellyfin desktop at http://localhost:8096/web/index.html#!/home.html or log in to Jellyfin on the Android app and look at next up in the home screen, the episode is missing.
Then I then looked at "http://localhost:8096/web/index.html#!/list.html?type=nextup&serverId=numbersredacted" on desktop and it is the correct next up episode.
However, if I watch a TV show on Desktop or Android, it correctly marks the episode as watched and adds the correct show on the Next Up home screen.
I've disabled plugins and tried this. Same events.    I've done multiple variations of testing on multiple platforms and different Roku versions. (Older models, Roku Smart TV, etc) Google Chrome desktop, Google Edge (Chrome version) desktop.
Thinking maybe something got left behind during the Roku app upgrade, I removed the Roku app, cleared my Roku cache, restarted Roku, reinstalled the Roku Jellyfin app and replicated this behavior.

To recap, if I watch a TV Show on Desktop Jellyfin or Android, it marks the TV show as watched and adds it to the Next Up section on the Jellyfin Home Screen across all platforms (Roku, Android, Desktop)

If I watch a TV Show on Roku 1.6.4, it marks it as watched, successfully adds the show as next up at http://localhost:8096/web/index.html#!/list.html?type=nextup, but removes it from the Next Up section of the home screen across all platforms (Desktop, Android, Roku)

**Comment by Avrution** [comment]
Same boat.  Also, I really dislike how now every first episode of a show appears on the "Next Up" list unless you set the max days to zero.  I have enjoyed having the list empty unless a show is started.
